master/internal/restore.go

```diff
@@ -134,79 +133,68 @@ func (m *Master) restoreExperiment(expModel *model.Experiment) error {
 	return nil
 }
 
-// restoreTrial takes the a searcher.Create and attempts to restore the trial that would be
-// associated with it. On failure, the trial is just reset to the start and errors are logged.
-func (e *internalExperiment) restoreTrial(
-	ckpt *model.Checkpoint, searcher experiment.TrialSearcherState,
+func (e *internalExperiment) restoreRun(
+	ckpt *model.Checkpoint, searcher experiment.RunSearcherState,
 ) {
-	l := e.syslog.WithField("request-id", searcher.Create.RequestID)
```

Comment:
why not include run-id as a field in your log here?

Reply:
because it could be nil at this point, a few lines down there's a nil check and if it exists, it is set on the syslog. 

question of whether it could be nil at this point is in the other comment